### DISCUSSION TITLE
Dataframe batcher use pre-existing model with overwrite disabled

### DISCUSSION THREAD

**Discussion by dpat** [gretelai/gretel-synthetics#115] [Q&A] [answered]
with Dataframe Batcher the current implementation requires a pre-existing checkpoints folder to be deleted, is there a method to generate records with the batcher using a pre-trained model? https://github.com/gretelai/gretel-synthetics/blob/master/src/gretel_synthetics/batch.py#L1022 it seems that this check should be a warning rather than preventing the use of previous checkpoints

**Comment by johntmyers** [comment, marked as answer]
@dpat If you set the `mode` to `READ` when you construct the Batcher, it will load an existing model that you can generate from. Is that what you are looking for?

**Reply by dpat** [reply]
This works thank you.
